backend/project/__init__.py

```diff
@@ -0,0 +1,34 @@
+#### BACKEND PROJECT PACKAGE INITIALIZATION ####
```

Comment:
```suggestion
"""BACKEND PROJECT PACKAGE INITIALIZATION"""
```
We should change this comment to a doc string so that the module will have a description whenever it is imported (in most IDEs/text editors).